client/app/organization/[id]/page.tsx

```
@@ -0,0 +1,629 @@
+                                    "Wednesday",
+                                    "Thursday",
+                                    "Friday",
+                                    "Saturday",
+                                  ][hour.day]
+                                }
+                              </span>
+                              <span className="font-mono">
+                                {hour.open} - {hour.close}
+                              </span>
+                            </div>
+                          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)